docker/CMSRucioClient/scripts/capacity_lucene.json

```diff
@@ -32,18 +32,11 @@
   "_source": {
     "includes": [
       "metadata.timestamp",
-      "metadata.kafka_timestamp",
       "data.name",
-      "data.disk_pledge",
-      "data.disk_usable",
       "data.disk_experiment_use",
-      "data.tape_pledge",
-      "data.tape_usable"
```

Comment:
Have you tested your changes with setQuotas? It looks OK since it looks like only disk_experiment_use is used, but should make sure.

Reply:
Yes, I did.
There is a bug in setQuotas though. I forgot to create an issue for that.

Reply:
It has not been running since whenever that change affected it.
Therefore I want to add some log processing for these scripts and send them to OpenSearch. Also, because ops wanted some visibility into our sync scripts.
Do you think it is good to do so? If yes, do you have any suggestions?